ddtrace/internal/runtime/runtime_metrics.py

```diff
@@ -55,23 +60,34 @@ class RuntimeWorker(_worker.PeriodicWorkerThread):
 
     FLUSH_INTERVAL = 10
 
-    def __init__(self, dogstatsd_url, flush_interval=None):
+    def __init__(self, tracer=None, dogstatsd_url=agent.get_stats_url(), flush_interval=None):
```

Comment:
```suggestion
    def __init__(self, tracer=None, dogstatsd_url=None, flush_interval=None):
```
I'd rather use `None` as the default and use that as a marker of the value being unset and then call the function.